kimchi/js/bindings.js

```diff
@@ -1410,7 +1404,7 @@ var caml_pasta_fp_plonk_index_read = function (offset, urs, path) {
   return plonk_wasm.caml_pasta_fp_plonk_index_read(
     offset,
     urs,
-    caml_jsstring_of_string(path),
+    caml_jsstring_of_string(path)
```

Comment:
no clear what's changing on these.  I guess it's just whitespace.

Reply:
Ah, that's something from prettier, from previous PR https://github.com/o1-labs/snarkyjs-bindings/pull/104.
I introduced with prettier the comma for the last parameter of a function, but jsoo is not happy with it, even though it seems to be accepted by some JS parsers.
It should not be there when rebased on develop.

Reply:
Yep, it's the trailing comma